tools/cdk-build-tools/bin/cdk-test.ts

```diff
@@ -70,12 +70,15 @@ async function main() {
     testCommand.push(args.nodeunit);
     testCommand.push(...testFiles.map(f => f.path));
 
-    await shell(testCommand, timers);
+    await shell(testCommand, {
+      timers,
+      env: { CDK_TEST_MODE: '1' }
```

Comment:
This concerns me. I actually rather we opt in from the toolkit to auto run the app on exit instead of opt out, or perhaps base it on the existence of CDK_OUT?